style.css

```diff
@@ -1,413 +1,892 @@
+    display: flex;
+    gap: 12px;
+    justify-content: flex-start;
+  }
+
+  .css {
+    color: rgba(9, 30, 66, 1);
+    padding: 12px 10px;
+    font-family: "Inter", sans-serif;
+    font-size: 15px;
+    border: 1px solid rgba(137, 147, 164, 1);
+    width: auto;
+  }
+
+  .invite-one {
+    font-family: "Inter", sans-serif;
+    font-size: 17px;
     text-align: center;
+    color: rgba(255, 255, 255, 1);
+    padding: 8px;
+  }
+}
+
+@media (min-width: 990px) {
+  .professional-art {
+    display: grid;
+    grid-template-columns: auto auto auto;
+    grid-gap: 10px;
+  }
+
+  .art-container.one,
+  .art-container.four {
+    background-image: url(icons/ImgPlaceholder1.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  .art-container.two,
+  .art-container.five {
+    background-image: url(icons/imgplace202.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  .art-container.three,
+  .art-container.six {
+    background-image: url(icons/imageplaceholder303.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  .art-container.one {
+    position: relative;
+  }
+
+  .art-container > .invite {
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    display: none;
+  }
+
+  .art-item {
+    background-image: none;
+    padding-top: 150px;
+  }
+
+  .data {
+    font-family: "Create Round", sans-serif;
+    font-size: 32px;
+    line-height: 44px;
+    font-weight: 400;
+    color: #fff;
+  }
+
+  .selections {
+    font-weight: 400;
+    font-family: "Inter", sans-serif;
+    font-size: 17px;
+  }
+
+  .art-item .buttonsli {
+    background-color: hsla(0, 0%, 50%, 0.068);
+    color: #fff;
+    font-weight: 500;
+    font-size: 15px;
+  }
+
+  .art-container {
+    height: 435px;
+  }
+
+  .art-container.one .invite {
+    display: block;
+  }
+}
+
+@media (min-width: 990px) {
+  .about-me {
+    display: flex;
+    flex-direction: column;
+    align-items: stretch;
+    padding: 100px;
+  }
+
+  .myresume {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    width: 50%;
   }
 
-  .desktop-navigation > ul > li + li {
-    margin-left: 32px;
+  .the-resume {
+    background-image: url(icons/illustration-about-me.png);
+    background-repeat: no-repeat;
+    background-position-x: 530px;
+    background-position-y: 0.47px;
   }
 
-  .main {
-    background-image: url("./images/Header-bg-desktop.png");
+  .h3-title {
+    font-family: "crete round", sans-serif;
+    font-weight: 400;
+    font-size: 72px;
   }
 
-  .main > .headline-section__box {
-    max-width: 920px;
-    margin: 0 auto;
+  .about-paragraph {
+    font-family: "Inter", sans-serif;
+    font-size: 20px;
+    font-weight: 400;
+    margin-bottom: 65px;
+  }
+
+  .resume {
+    font-weight: 700;
+    font-size: 17px;
+    padding: 12px;
+  }
+
+  .work-on {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
```

Comment:
@mesfin345 the project is complet ! congratulations 👏 

You have done a good job adding the changes suggested by the previous reviewer !